Check for rt library in configuration -- support for glibc<2.17 (#2854)
CVC4/CVC4: src/CMakeLists.txt

target_link_libraries(cvc4 ${GMP_LIBRARIES})
target_include_directories(cvc4 PUBLIC ${GMP_INCLUDE_DIR})

> # Add rt library

Reviewer: ```suggestion
```

> # Note: for glibc >= 2.17 this is not necessary because rt is already included

Reviewer: ```suggestion
# Note: For glibc < 2.17 we have to additionally link against rt (man clock_gettime).
```

> #       in that case RT_LIBRARIES will be empty

Reviewer: ```suggestion
```
